shaders/lang/ru_RU.lang

```diff
@@ -61,684 +120,1432 @@ screen.Direct_Light = Прямой свет
+
+[-----Max Shadow Bounds-----]
+
+        option.OPTIMIZED_SHADOW_DISTANCE = Максимальные Грани Теней
+            value.OPTIMIZED_SHADOW_DISTANCE.-1.0 = Не-Оптимизированные
+            value.OPTIMIZED_SHADOW_DISTANCE.1.0 = Оптимизированные
+
+[-----Entity Shadow Distance-----]
+
+        option.entityShadowDistanceMul = Дистанция Теней Сущности
+            value.entityShadowDistanceMul.0.01 = 1% от Дистанции Теней
+            value.entityShadowDistanceMul.0.02 = 2% от Дистанции Теней
+            value.entityShadowDistanceMul.0.03 = 3% от Дистанции Теней
+            value.entityShadowDistanceMul.0.04 = 4% от Дистанции Теней
+            value.entityShadowDistanceMul.0.05 = 5% от Дистанции Теней
+            value.entityShadowDistanceMul.0.10 = 10% от Дистанции Теней
+            value.entityShadowDistanceMul.0.15 = 15% от Дистанции Теней
+            value.entityShadowDistanceMul.0.20 = 20% от Дистанции Теней
+            value.entityShadowDistanceMul.0.25 = 25% от Дистанции Теней
+            value.entityShadowDistanceMul.0.30 = 30% от Дистанции Теней
+            value.entityShadowDistanceMul.0.35 = 35% от Дистанции Теней
+            value.entityShadowDistanceMul.0.40 = 40% от Дистанции Теней
+            value.entityShadowDistanceMul.0.45 = 45% от Дистанции Теней
+            value.entityShadowDistanceMul.0.50 = 50% от Дистанции Теней
+            value.entityShadowDistanceMul.0.75 = 75% от Дистанции Теней
+            value.entityShadowDistanceMul.1.00 = 100% от Дистанции Теней
+
+|/////Shadow Filtering Settings/////|
+
+    screen.Filtering = Настройка Фильтра Теней
+
+[-----Basic Shadow Filter Samples-----]
+
+        option.SHADOW_FILTER_SAMPLE_COUNT = Обычный Образец Фильтра Теней
+
+[-----Minimum Shadow filter Radius-----]
+
+        option.Min_Shadow_Filter_Radius = Минимальный Радиус Фильтра Теней
+
+[-----Variable Penumbra Shadows-----]
+
+        option.Variable_Penumbra_Shadows = Переменные Полутеневые Тени
+
+[-----VPS Filter Samples-----]
+
+        option.VPS_Search_Samples = Образцы Фильтра ППТ (VPS)
+
+[-----Maximum Shadow Filter Radius-----]
+
+        option.Max_Shadow_Filter_Radius = Максимальный Радиус Фильтра Теней
+
+[-----Sun Size Factor-----]
+
+        option.Max_Filter_Depth = Фактор Размера Солнца
+
+|/////FloodFill/////|
+
+    screen.LPV = НаливноеЗаполнение (FloodFill) - §cОБЯЗАТЕЛЬНО IRIS§r
+
+[-----Enabled-----]
+
+        option.LPV_ENABLED = Включён
+
+[-----Size-----]
+
+        option.LPV_SIZE = Размер
+            value.LPV_SIZE.6 = Маленький [64]
+            value.LPV_SIZE.7 = Средний [128]
+            value.LPV_SIZE.8 = Большой [256]
+
+[-----Light Saturation-----]
+
+        option.LPV_SATURATION = Насыщенность Света
+            suffix.LPV_SATURATION=%
+
+[-----Tint Saturation-----]
+
+        option.LPV_TINT_SATURATION = Насыщенность Тонировки (Tint)
+            suffix.LPV_TINT_SATURATION=%
+
+[-----Normal Strength-----]
+
+        option.LPV_NORMAL_STRENGTH = Сила Нормалей
```

Comment:
```suggestion
        option.LPV_NORMAL_STRENGTH = Сила Карты Нормалей
```

Reply:
I hope I'm not blind and really see that Normal Strength doesn't have the word “Map” in it so it's weird to call it that.
I already wrote about it in the beginning.